### DISCUSSION TITLE
(commits) I have done today is being shown in yesterday's activity

### DISCUSSION THREAD

**Discussion by Aliyan-Ansari** [community/community#61351] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Today, I committed and pushed my code, but in the progress table, today's commits are being shown under yesterday's commits, and today's progress is empty.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by UmerGillani36** [comment, marked as answer]
you need to set your date and time because it reflects on github from you machine time

**Comment by AteebNoOne** [comment]
Answer should be updated now!
